lib/spack/spack/cmd/find.py

```diff
@@ -342,10 +355,20 @@ def find(parser, args):
 
         count_suffix = " (not shown)"
         if not args.only_roots:
-            cmd.display_specs(results, args, decorator=decorator, all_headers=True)
             count_suffix = ""
+            kwargs = {"decorator": decorator, "all_headers": True, "status_fn": status_fn}
+            cmd.display_specs(results, args, **kwargs)
 
         # print number of installed packages last (as the list may be long)
         if sys.stdout.isatty() and args.groups:
+            if args.show_concretized:
+                spack.cmd.print_how_many_pkgs(
+                    list(x for x in results if not x.installed),
+                    "concretized-but-not-installed",
+                    suffix=count_suffix,
+                )
```

Comment:
see my other comment -- I think this needs to be reworded (like @haampie said) and that it should hint that you can use `--show-concretized` to see them.

Reply:
Harmen didn't want this part at all, regardless of wording. It sounds like you want this, but worded differently. I will change the wording.

Reply:
Still interested in whether @haampie likes my suggestion but yes this is what I was suggesting.